src/firefly/js/metaConvert/WiseRequestList.js

```diff
@@ -47,6 +50,16 @@ export function makeWisePlotRequest(table, row, includeSingle, includeStandard,
                 req.setParam('subsize', `${newSize}`);
             }
         }
+        /*
+        if (table.request.table_name.includes('3band')||table.request.table_name.includes('2band')) {
+            const tblBand = table.request.table_name.includes('3band') ? 3 : 2;
+
+            // add note to replace the query returned fail reason 'No Found'
+            if (Number(extraParams.band) > tblBand) {
+                req.setParam('userFailReason', {'not found': 'No image for band ' + extraParams.band});
+            }
```

Comment:
it was made to give some supplemental message  in case the returned message is 'not found' for band 2 and band 3  request. However, it was determined later to fix the request for band 2 and band 3 cases. Therefore, this part was commented and kept in case it is needed in the future.